src/components/events/commonEventFields/eventLists.tsx

```diff
@@ -1,23 +1,23 @@
 import { Typography } from '@mui/material';
-import { eventKeyDisplay, eventLists } from '../../../types/configurations';
+import { EVENT_KEY_DISPLAY, EVENT_LISTS } from '../../../types/configurations';
 import { IEvent } from '../../../types/interfaces';
 import ListGenerator from '../../general/listGenerator';
 import List from '../../general/list';
 
-interface ListsProps {
+interface Props {
     event: IEvent;
     editable: boolean;
     setAttr: (attr: keyof IEvent, value: any) => void;
 }
 
-const EventLists = ({ event, setAttr, editable }: ListsProps) => (
+const EventLists = ({ event, setAttr, editable }: Props) => (
   <>
-    {eventLists.map((list) => (
+    {EVENT_LISTS.map((list) => (
       (editable && setAttr)
         ? (
           <ListGenerator
             header={
-              eventKeyDisplay.find((k) => k.key === list)?.display || 'שגיאה'
+              EVENT_KEY_DISPLAY.find((k) => k.key === list)?.display || 'שגיאה'
             }
             attr={list as keyof IEvent}
```

Comment:
EVENT_LISTS is in type  keyof IEvent ? 
if so .. use it

Reply:
Done